src/main/resources/reference.conf

```diff
@@ -167,6 +167,12 @@ sparkz {
     # Limit for number of modifiers to request and process at once
     maxRequestedPerPeer = 1024
 
+    # Enables or disables slow mode - ignoring requesting or broadcasting new transaction when node is overloaded
+    slowModeFeatureFlag = false
+
+    # Threshold of average time it takes node to apply a new modifier, after which we consider it overloaded and start throttling
```

Comment:
let's extend the comment to specify that we consider both blocks and tx modifiers